.golangci.goheader.template

```diff
@@ -0,0 +1,13 @@
+Copyright © {{ copyright-year }} Meroxa, Inc.
```

Comment:
where is this being injected?

Reply:
this go header file is referenced for linting check in .golangci.yml file